api/environments/identities/views.py

```diff
@@ -1,15 +1,22 @@
+import base64
+import json
+import typing
 from collections import namedtuple
 
 import coreapi
+from boto3.dynamodb.conditions import Key
 from rest_framework import status, viewsets
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.schemas import AutoSchema
 
-from app.pagination import CustomPagination
+from app.pagination import CustomPagination, EdgeIdentityPagination
 from environments.identities.helpers import identify_integrations
-from environments.identities.models import Identity
-from environments.identities.serializers import IdentitySerializer
+from environments.identities.models import Identity, dynamo_identity_wrapper
```

Comment:
Yeah, this isn't right, we shouldn't be passing the `dynamo_identity_wrapper` object around between modules like this. Let's move all the logic that uses the wrapper into the `models` module using methods on the `Identity` class (staticmethod, classmethod or otherwise)

Reply:
something like
```

@python_2_unicode_compatible
class Identity(models.Model):
    identifier = models.CharField(max_length=2000)
    created_date = models.DateTimeField("DateCreated", auto_now_add=True)
    environment = models.ForeignKey(
        Environment, related_name="identities", on_delete=models.CASCADE
    )

    dynamo_identity_wrapper = DynamoIdentityWrapper()



```

which makes the interface looks something like

```
        return Identity.dynamo_identity_wrapper.get_item_from_uuid(
            self.kwargs["environment_api_key"], self.kwargs["identity_uuid"]
        )



```